package.ps1

```diff
@@ -0,0 +1,31 @@
+﻿#This file is from UMM Derail Valley Template 
```

Comment:
I will probably never use this script, as I use WSL and Bourne shell for my own release automation.

Reply:
Script has been removed.